toolbox/repo/validate_role_vars_used.py

```diff
@@ -0,0 +1,78 @@
+#! /usr/bin/env python
+
+import glob, os
+import yaml
+import subprocess
+import pathlib
+
+THIS_DIR = pathlib.Path(__file__).absolute().parent
+
+# go to top directory
+os.chdir(os.path.join(THIS_DIR, "..", ".."))
```

Comment:
Inconsistent with the other script (other script was nicer IMO)

Reply:
👍🏻 I remembered seing a Py file of yours using it after writing the scripts; I didn't know this library and it's quite nice :)